src/redux/actions/profile/__test__/handle-update-profile-error.spec.js

```diff
@@ -0,0 +1,20 @@
+import handleUpdateProfileError from '../handle-update-profile-error';
+
+jest.mock('../../app/show-error-message', () => jest.fn(error => ({
+    type: 'SHOW_ERROR_MESSAGE',
+    message: error.message,
+})));
```

Comment:
```jest.mock('../../app/show-error-message', () => jest.fn());```
import showErrorMessage and only test that showErrorMessage is called
also you need to test that dispatch is called two times